packages/i18n/src/messages/en.json

```
@@ -2678,15 +2683,38 @@
     "callControlsScreenshare": "Screen share",
     "callControlsScreenshareActiveAria": "Screen share, active",
     "callControlsScreenshareInactiveAria": "Screen share, inactive",
+    "callReactButton": "React",
```

Biome (2.4.16)

[error] The key callReactButton was already declared.

(lint/suspicious/noDuplicateObjectKeys)